docs/reference/cli.md

```diff
@@ -0,0 +1,194 @@
+---
```

Comment:
@hubvu should we move this content into the _index..md file instead? The link would be https://docs.ondat.io/docs/reference/cli/ as well

Reply:
hey @cannischan - looks like that won't be possible due to an existing `_index.md` file in the reference directory which lists the rest of the reference pages.

```yaml
.
├── _index.md
├── api.md
├── bundles
│   ├── _index.md
│   ├── diagnostic-bundle.md
│   └── support_bundle.md
├── cli.md
├── cluster-operator
│   ├── _index.md
│   ├── configuration.md
│   ├── examples.md
│   ├── install.md
│   └── upgrade.md
├── clusters.md
├── contributing.md
├── filesystems.md
├── gui.md
├── init-container.md
├── kubectl-plugin.md
├── kubernetes-object-sync.md
├── namespaces.md
├── open_source_attribution.md
├── policies.md
├── resource_requests_and_limits.md
├── scheduler
│   ├── _index.md
│   └── admission-controller.md
├── telemetry.md
└── tolerations.md

3 directories, 26 files
```